### PR TITLE
General documentation maintenance

### PR DESCRIPTION
**Pull request by djmattyg007**
- Update some URLs to use HTTPS, or to follow redirects
- Account for the switch from setup.py to setup.cfg
- Update notes about installing graphviz to account for the fact that
  pygraphviz is now a docs dependency defined in setup.cfg

### CODE REVIEW

File: docs/devenv.rst [hunk 1]
```diff
@@ -359,15 +359,17 @@ If successful, the command will not print anything at all.
...
-To write documentation, we use `Sphinx <http://www.sphinx-doc.org/>`_. See
+To write documentation, we use `Sphinx <https://www.sphinx-doc.org/>`_. See
 their site for lots of documentation on how to use Sphinx.
 
 .. note::
 
-    To generate a few graphs which are part of the documentation, you need some
-    additional dependencies. You can install them from APT with::
+    To generate a few graphs which are part of the documentation, you need to
+    install the application 'graphviz'. You can install them from APT with::
```
Comment (kingosticks) [inline docs/devenv.rst:368]:
```suggestion
    install the application 'graphviz'. You can install it from APT with::
```

**Review by kingosticks — APPROVED** [review]

File: docs/extensiondev.rst [hunk 2]
```diff
@@ -87,7 +87,7 @@ with ``#egg=Mopidy-Something-dev`` for installation using
     Mopidy-Soundspot
     ****************
 
-    `Mopidy <http://www.mopidy.com/>`_ extension for playing music from
+    `Mopidy <https://www.mopidy.com/>`_ extension for playing music from
```
Comment (jodal) [inline docs/extensiondev.rst:90]: `www.mopidy.com` actually redirects to `mopidy.com`.
```suggestion
    `Mopidy <https://mopidy.com/>`_ extension for playing music from
```

**Review by jodal — APPROVED** [review]

**Comment by djmattyg007** [comment]
Thanks for the review :) I've force-pushed the PR branch to include the updates everyone recommended.

File: docs/devenv.rst [hunk 3]
```diff
@@ -359,15 +359,17 @@ If successful, the command will not print anything at all.
...
-To write documentation, we use `Sphinx <http://www.sphinx-doc.org/>`_. See
+To write documentation, we use `Sphinx <https://www.sphinx-doc.org/>`_. See
 their site for lots of documentation on how to use Sphinx.
 
 .. note::
 
-    To generate a few graphs which are part of the documentation, you need some
-    additional dependencies. You can install them from APT with::
+    To generate a few graphs which are part of the documentation, you need to
+    install the graphviz package. You can install them from APT with::
```
Comment (kingosticks) [inline docs/devenv.rst:368]:
```suggestion
    install the graphviz package. You can install it from APT with::
```
You missed this one!

Comment (djmattyg007) [reply docs/devenv.rst:368]: Oops!

Comment (djmattyg007) [reply docs/devenv.rst]: Fixed. Thank you!

**Pull request merged**